types/src/v19/blockchain/into.rs

```diff
@@ -124,6 +124,7 @@ impl MempoolEntry {
     pub fn into_model(self) -> Result<model::MempoolEntry, MempoolEntryError> {
         use MempoolEntryError as E;
 
+        let vsize = Some(crate::to_u32(self.vsize, "size")?);
```

Comment:
```suggestion
        let vsize = Some(crate::to_u32(self.vsize, "vsize")?);
```

The error string is the field name (by convention).